src/Image.php

```diff
@@ -64,7 +64,7 @@ public function lazyloadBackgroundImages( $html, $buffer ) {
 				continue;
 			}
 
-			$url['url'] = trim( $url['url'], '\'" ' );
+            $url['url'] = esc_url( trim( html_entity_decode($url['url'], ENT_QUOTES), '\'" ' ) );
```

Comment:
@engahmeds3ed the spacing seems off. Do you see the same issue? Also please make sure you use tab spacing instead of spaces